src/vxingest/main.py

```diff
@@ -202,37 +206,98 @@ def create_dirs(paths: list[Path]) -> None:
         path.mkdir(parents=True, exist_ok=True)
 
 
-class JobDoc(TypedDict):
-    """A type class describing the expected return result from Couchbase"""
+class JobRunCriterion(TypedDict):
+    """A type class describing the criterion for running a job, and the
+    expected return result from Couchbase"""
 
     id: str
     name: str
     offset_minutes: int
     run_priority: int
-    sub_type: str
+    subType: str
 
 
-def get_job_docs(
+def get_runtime_job_criteria(
     cluster: Cluster,
     creds: dict[str, str],
-    job_id: Optional[str] = None,
-) -> list[JobDoc]:
-    """Queries Couchbase for the given job doc or job docs in need of processing if no job ID is given"""
+    job_id: str | None = None,
+) -> JobRunCriterion | None:
+    """
+    Queries the Couchbase database for a specific job document by its ID.
+    This is used to retrieve the newer runtime job documents type "JS" for processing.
+
+    Args:
+        cluster (Cluster): The Couchbase cluster instance to use for querying.
+        creds (dict[str, str]): A dictionary containing Couchbase credentials, including 'cb_bucket' and 'cb_scope'.
+        job_id str | None = None, optional): The ID of the job document to retrieve. Must be provided.
 
-    # TODO - We're doing this query at the cluster level. Would it be better to query at the scope level if we're using Couchbase 7?
-    # https://docs.couchbase.com/python-sdk/current/howtos/n1ql-queries-with-sdk.html#querying-at-scope-level
+    Returns:
+        processDoc | None: The job document if found, otherwise None.
+
+    Raises:
+        ValueError: If job_id is not provided.
+    """
+    if not job_id:
```

Comment:
I'm not sure if this method operates on a `JobDoc`, `JobSpec`, or `ProcessSpec`. Can you clarify the naming? Using `job_id` is a bit confusing. 

I'm also curious why this method is so much shorter than the original `get_job_docs()`.

After looking at the old method - I'm curious if this could be named `get_job_spec()`.